apps/challenges/models.py

```diff
@@ -12,6 +14,11 @@
 class Challenge(TimeStampedModel):
 
     """Model representing a hosted Challenge"""
+
+    def __init__(self, *args, **kwargs):
+        super(Challenge, self).__init__(*args, **kwargs)
+        self.__original_evaluation_script = self.evaluation_script
```

Comment:
@deshraj @TroJan Actually, I used that to be specific about the class private member names. Please refer [here](http://stackoverflow.com/questions/1301346/what-is-the-meaning-of-a-single-and-a-double-underscore-before-an-object-name). Moreover, I didn't know about the convention we have been following. I will update the PR accordingly.